dist/image-sequencer.js

```diff
@@ -47721,6 +47721,25 @@ ImageSequencer = function ImageSequencer(options) {
     else modulesdata = modules[name][1];
     return modulesdata;
   }
+
+  function toString(step_name,options){
+    if(step_name){
+      return `${step_name}(${Object.keys(options).map(key=>key + ':' + options[key]).join(',')})`
+    }
+    else{
+      return  copy(this.images.image1.steps).map(step=>{
```

Comment:
And here, while ES6 is great, shall we stick to a consistently styled `function step() {` here just to remain consistent?